packages/material-ui/src/styles/experimentalStyled.test.js

```diff
@@ -56,6 +57,26 @@ describe('experimentalStyled', () => {
     });
   });
 
+  it('REMOVE THIS: emotion theme should work', () => {
```

Comment:
This test is failing :\ I don't understand what I am doing wrong...

Reply:
React theme context singleton issue, theme-ui force v10 when we need v11:

```diff
diff --git a/package.json b/package.json
index ae5142d8dd..ccc26e9c4b 100644
--- a/package.json
+++ b/package.json
@@ -173,6 +173,7 @@
     "**/@babel/preset-typescript": "^7.10.4",
     "**/@babel/runtime": "^7.10.2",
     "**/@babel/types": "^7.10.2",
+    "**/@emotion/styled": "^11.0.0",
     "**/cross-fetch": "^3.0.5",
     "**/dot-prop": "^5.2.0",
     "**/react-is": "^16.13.1",
```